Source/WebKit/Platform/IPC/Connection.h

```diff
@@ -398,6 +406,12 @@ class Connection : public ThreadSafeRefCounted<Connection, WTF::DestructionThrea
     std::unique_ptr<Encoder> m_pendingWriteEncoder;
     EventListener m_writeListener;
     HANDLE m_connectionPipe { INVALID_HANDLE_VALUE };
+#elif PLATFORM(HAIKU)
+	Identifier m_connectedProcess;
```

Comment:
Whats the identation to be followed this is not how how it looks in Pe it looks rightly idented but here it looks differently

Reply:
I would recommend Koder instead of Pe then! I think it can show tabs and spaces.
Unfortunately the coding style is a mixup of both, so try to stick to whatever is used in the file you're editing, for now.